src/include/cell_inline.h

```diff
@@ -662,11 +662,11 @@ __wt_cell_leaf_value_parse(WT_PAGE *page, WT_CELL *cell)
  * case, make sure all reads are inside the page image. If an error occurs, return an error code but
  * don't output messages, our caller handles that.
  */
-#define WT_CELL_LEN_CHK(t, len)                                                             \
```

Comment:
Could we use a better name for `t`? It's the start of the range being checked. Let me know if you'd like a more concrete suggestion...